### ISSUE TITLE
Name in Experian personal details form is the wrong heading

### ISSUE THREAD

**Issue by emmastephenson** [CDCgov/prime-simplereport#4077]
## Area or Component
Sign up flow - Experian personal details form

## Quick Summary
The person's name is an `<h3>` element, but this is semantically out of order (there's no h2 between the h1 of "Step2of3" and the name element)

## Accessibility Violation
Heading levels should only increase by one: https://dequeuniversity.com/rules/axe/4.4/heading-order?application=AxeChrome

## Action Requested
Change the name to an h2 element instead of h3. If desired, use CSS to keep the styling the same.

## Screenshots (optional)
(The name here is "Water Bottle"...running out of creative naming energy over here)
![Screen Shot 2022-07-26 at 3 24 41 PM](https://user-images.githubusercontent.com/80282552/181122706-7074c86d-026a-46d3-b481-1bb41e9f6566.png)

## Original Deque Issue
https://docs.google.com/spreadsheets/d/1b6xjubAN5Uw11DV4_B5zvSu7HeDI6Z-DpV4DMcToqAY/edit#gid=543263785&range=92:92
